src/appleseed.shared/application/progresstilecallback.h

```diff
@@ -29,23 +29,23 @@
 
 #pragma once
 
+// appleseed.shared headers
+#include "dllsymbol.h"
+
 // appleseed.renderer headers.
 #include "renderer/api/rendering.h"
 
 // Standard headers.
 #include <memory>
 
-// Forward declarations.
-namespace foundation    { class Logger; }
-
 namespace appleseed {
-namespace cli {
+namespace shared {
 
-class ProgressTileCallbackFactory
+class SHAREDDLL ProgressTileCallbackFactory
```

Comment:
If you expose this class, then it cannot use anything from the `std::` namespace. You'll need to provide a private implementation.

Reply:
Thanks for the review @dictoon!
I'll make the changes ASAP.